.buildkite/publish-docs.sh

```diff
@@ -3,18 +3,16 @@ set -eo pipefail
 
 source .buildkite/install-repo.sh
 
-echo --- Initializing gcloud
-
-
 echo --- Building docs
 pushd docs
 EXIT_CODE=0
-pdm run make deploy || EXIT_CODE=$?
+PDM=${PDM_COMMAND:1:-1} ${PDM_COMMAND:1:-1} run make deploy  || EXIT_CODE=$?
```

Comment:
An ignorant question: what does ${PDM_COMMAND:1:-1} do? I thought it produced a substring with the first and last characters removed

Reply:
It does! Buildkite has the weird side-effect of hard-quoting all env variables set in the pipeline.yml (and I don't want to search replace in a million files for pdm25 whenever we upgrade). So if we try to run `$PDM_COMMAND` it actually runs `"pdm25" some command`, but `"pdm25"` isn't a program - only `pdm25` is.